pkgs/desktops/gnome-3/extensions/pop-shell/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, fetchFromGitHub, glib, nodePackages }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-pop-shell";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "shell";
+    rev = version;
+    sha256 = "0nws28w40xpwhgdpcifzys1md4h6a7q5k3wcn2a77z5m26g69zw2";
+  };
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+  buildInputs = [ glib nodePackages.typescript ];
+
+  postInstall = ''
+    mv $out/usr/* $out
+    rmdir $out/usr
+  '';
+
+  meta = with stdenv.lib; {
+    description = ''
```

Comment:
That is the long description. A short one is also needed.

Reply:
Added